openlibrary/plugins/openlibrary/home.py

```diff
@@ -2,23 +2,18 @@
 """
 import random
 import web
-import simplejson
 import logging
 
 from infogami.utils import delegate
 from infogami.utils.view import render_template, public
 from infogami.infobase.client import storify
-from infogami import config
```

Comment:
This might break things; I think importing the config has side-effects, so although it's not explicitly used it impacts the code.

Reply:
That sounds like exceedingly sketchy behavior, but duly noted.